srv/PutInformJob.srv

```diff
@@ -0,0 +1,25 @@
+# SPDX-FileCopyrightText: 2024, Yaskawa America, Inc.
+# SPDX-FileCopyrightText: 2024, Delft University of Technology
+#
+# SPDX-License-Identifier: Apache-2.0
+
+# NOTE:
+# - do not include the file extension (JBI), only the name of the job
+# - character encodings other than ASCII are only partially supported
+# - a job with this name must NOT exist on the controller
+string name
```

Comment:
It seems this field is not needed and can be removed. The controller will take the name in the job header. The only time this field is used is to create the temporary file that contains the job contents, which after the job is loaded gets removed.